### PR TITLE
Extend pretty-printer package to pretty-print packages

### PR DESCRIPTION
**Pull request by StachuDotNet**
Changelog:

```
packages/pretty-printer
- (incomplete) extend to pretty-print packages
```

### CODE REVIEW

File: packages/darklang/stdlib/canvas.dark [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module Darklang =
```
Comment (pbiggar) [inline packages/darklang/stdlib/canvas.dark:1]: I don't think this would go in the stdlib package.

Comment (pbiggar) [reply packages/darklang/stdlib/canvas.dark:1]: Maybe "language tools" or similar as a package name?

File: scripts/build/compile [hunk 2]
```diff
@@ -353,6 +353,9 @@ def mark(should, f):
   elif ("/backend/" in f) and ("/tests/" in f):
     should.backend_test = True
 
+  elif ("/packages/" in f) and (f.endswith(".dark")):
+    should.backend_quick_build = True
```
Comment (pbiggar) [inline scripts/build/compile:357]: What's the goal here?

Comment (StachuDotNet) [reply scripts/build/compile:357]: parse and re-load packages upon save of a package

might be worth
- extracting "load packages" from the run-backend-server script
- add a `should.reload_packages`
- when doing any backend build, set that^ to `true`
?

Comment (pbiggar) [reply scripts/build/compile:357]: Yeah, thats a better (and faster) approach

**Pull request merged**